frontend/src/js/controllers/challengeListCtrl.js

```diff
@@ -53,6 +53,19 @@
                     vm.challengeCreator[id]= vm.currentList[i].creator.id;
                     utilities.storeData("challengeCreator", vm.challengeCreator);
                 }
+                if(vm.currentList.length !=0){
+                     vm.presentLoadIndex=3;
+                     vm.showMore = function() {
+                        if (vm.presentLoadIndex < vm.currentList.length){
+                           vm.presentLoadIndex = vm.presentLoadIndex+1;
+                        }
+                     };
+                     vm.showLess = function() {
```

Comment:
Why there is a check of 3 @Obtin, we are already sending pagination limit in API. Please check response.

Reply:
Done